sdk/deviceupdate/azure-iot-deviceupdate/src/test/java/com/azure/iot/deviceupdate/TestData.java

```diff
@@ -10,7 +10,7 @@ public class TestData {
     public static final String CLIENT_ID = getTestValue("CLIENT_ID", "clientId");
 
     public static final String ACCOUNT_ENDPOINT = getTestValue("ACCOUNT_ENDPOINT",
-        "contosoprod.api.prod.adu.microsoft.com");
+        "https://adu-sdk-testing.api.adu.microsoft.com");
```

Comment:
why this change?

Reply:
because endpoint needs to be a url (start with http) so that it can concat with relative url.

Reply:
then you are wrong here.

see the host is composed based on the endpoint https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/deviceupdate/azure-iot-deviceupdate/src/main/java/com/azure/iot/deviceupdate/implementation/DeviceManagementsImpl.java#L71

We might need to think how we should handle this in codegen on full-auto. But here apparently you should have `"https://" + this.client.getEndpoint()`